src/frontend/src/lib/components/send/SendTokensList.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	import { BigNumber } from '@ethersproject/bignumber';
+	import Listener from '$lib/components/core/Listener.svelte';
+	import TokensSkeletons from '$lib/components/tokens/TokensSkeletons.svelte';
+	import { enabledNetworkTokensUi } from '$lib/derived/network-tokens.derived';
+	import type { TokenUi } from '$lib/types/token';
+	import { fade } from 'svelte/transition';
+	import { formatToken } from '$lib/utils/format.utils';
+	import CardAmount from '$lib/components/ui/CardAmount.svelte';
+	import ExchangeTokenValue from '$lib/components/exchange/ExchangeTokenValue.svelte';
+	import { balancesStore } from '$lib/stores/balances.store';
+	import { createEventDispatcher } from 'svelte';
+	import TokenCardWithOnClick from '$lib/components/tokens/TokenCardWithOnClick.svelte';
+
+	const dispatch = createEventDispatcher();
+
+	let tokens: TokenUi[];
+	$: tokens = $enabledNetworkTokensUi.filter(({ id: tokenId }) =>
+		($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n)
+	);
+</script>
+
+<TokensSkeletons>
+	{#each tokens as token (token.id)}
+		<Listener {token}>
+			<div in:fade>
```

Comment:
Do we need `fade` ? There is no effect no?

Reply:
this is just precautious in case the user clicks on the button so quickly that the tokens are still not loaded. Shall I remove it?

Reply:
If we want to be consistent then yes we remove the fade and in addition we gonna have to use `waitWalletReady` when the modal is opened to wait until the tokens are not loaded.

Reply:
ah! but then, we should do this in `TokensSignedIn` too, I image? In another PR

Reply:
Ah! I understand now! Yes, I had already put it [there](https://github.com/dfinity/oisy-wallet/pull/1877/files#diff-3eaedffe3c4906283cba5482203936b32f7e6ff2997ab193db4956f4b3380410).